### PR TITLE
Adds Postgres 14 docker image and moves existing on into postgres 13

### PR DESCRIPTION
**Pull request by gurkanindibay**
I moved existing Docker image into postgres-13 and changed latest docker image into pg 14 supported images

### CODE REVIEW

File: .github/workflows/publish_docker_images_cron.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone -b v0.8.1 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b pg-14-docker --depth=1  https://github.com/citusdata/tools.git tools
```
Comment (gurkanindibay) [inline .github/workflows/publish_docker_images_cron.yml:29]: After approval of https://github.com/citusdata/tools/pull/182 and tag creation, branch name will be replaced

**Comment by hanefi** [comment]
@gurkanindibay Can we also re-enable hll and topn with updated versions that actually support PG14 ?

**Comment by gurkanindibay** [comment]
> ctually support PG14

Postgresql-14-hll and topn packages does not exist. Since they are not required and docker is urgent I think we can fix this with further PR

**Review by hanefi — APPROVED** [review]

File: .github/workflows/publish_docker_images_cron.yml (hunk 1 above)
Comment (hanefi) [reply .github/workflows/publish_docker_images_cron.yml:30]: Please replace the tag name, and update the line here.

File: .github/workflows/publish_docker_images_on_manual.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone -b v0.8.1 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b pg-14-docker --depth=1  https://github.com/citusdata/tools.git tools
```
Comment (hanefi) [inline .github/workflows/publish_docker_images_on_manual.yml:29]: needs update

File: .github/workflows/publish_docker_images_on_push.yml [hunk 3]
```diff
@@ -20,13 +20,14 @@ jobs:
...
           - postgres_12
+          - postgres_13
           - nightly
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone -b v0.8.1 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b pg-14-docker --depth=1  https://github.com/citusdata/tools.git tools
```
Comment (hanefi) [inline .github/workflows/publish_docker_images_on_push.yml:30]: needs update

Comment (gurkanindibay) [reply .github/workflows/publish_docker_images_on_push.yml]: Fixed

File: .github/workflows/publish_docker_images_on_manual.yml (hunk 2 above)
Comment (gurkanindibay) [reply .github/workflows/publish_docker_images_on_manual.yml]: Fixed

File: .github/workflows/publish_docker_images_cron.yml (hunk 1 above)
Comment (gurkanindibay) [reply .github/workflows/publish_docker_images_cron.yml]: Fixed

**Pull request merged**